### ISSUE TITLE
 v1.3.1 Cannot use 'new' with an expression whose type lacks a call or construct signature

### ISSUE THREAD

**Issue by TaylorPzreal** [quilljs/quill#1634]
When i upgrade ```quill@1.3.0``` , ```@types/quill@0.0.31``` to ```quill@1.3.1```,```@types/quill@1.3.0```, i got this error.
```javascript
import 'quill/dist/quill.snow.css';
import * as Quill from 'quill';

    const editor: any = new Quill('#quill-editor', {
      bounds: '#quill-editor',
      modules: {
        // formula: true,
        // syntax: true,
        toolbar: this.toolbarOptions,
        // history: {
        //   delay: 2000,
        //   maxStack: 500,
        //   userOnly: true
        // }
      },
      placeholder: 'Write...',
      theme: 'snow'
    });
```

My development environment:
angular@4.3.3
webpack@3.4.1

**Expected behavior**:
no errors

**Actual behavior**:
 Cannot use 'new' with an expression whose type lacks a call or construct signature

**Platforms**:
angular@4.3.3
webpack@3.4.1
quill@1.3.1
types/quill@1.3.0

Linux Mint 18, Chrome@latest

**Version**:
1.3.1

**Comment by benbro** [comment]
What are the steps to reproduce?

Do you see the error on the [CodePen](https://codepen.io/quill/pen/KzZqZx)?

**Comment by benbro** [comment]
The syntax of ```types/quill"` changed to
```import Quill from 'quill';```
but I'm still getting an error when using ```types/quill@1.3.2"
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/18947

**Comment by benbro** [comment]
Fixed in [v2.0.0-beta.2](https://github.com/quilljs/quill/releases/tag/v2.0.0-beta.2)
